pkg/controllers/vrep/replication_reconciler.go

```diff
@@ -225,6 +225,13 @@ func (r *ReplicationReconciler) collectPodFacts(ctx context.Context) (err error)
 // choose the source host and target host
 // (first host where db is up in the specified cluster)
 func (r *ReplicationReconciler) determineSourceAndTargetHosts() (err error) {
+	if len(r.SourcePFacts.Detail) == 0 && r.TargetPFacts.SandboxName != "" {
```

Comment:
```suggestion
	if len(r.SourcePFacts.Detail) == 0 && r.TargetPFacts.SandboxName != vapi.MainCluster {
```
minor: a const we use and that is equal to empty string